src/library_async.js

```diff
@@ -168,7 +173,7 @@ mergeInto(LibraryManager.library, {
             original = new WebAssembly.Function(
               { parameters , results: ['externref'] },
               original,
-              { promising : 'first' });
+              { 'promising' : 'first' });
```

Comment:
Is this an unrelated fix?

Reply:
The new test that is enabled runs with closure and closure will rename this if it isn't in quotes.

Reply:
I see... what about the first argument above though?  Why does that get away without quotes?  I wonder if there is something we you modify in the closure extern files to avoid the need for this?